src/nu/validator/xml/WiretapXMLReaderWrapper.java

```diff
@@ -60,13 +60,13 @@ public WiretapXMLReaderWrapper(final XMLReader wrappedReader) {
     }
 
     /**
-     * Sets the wiretapContentHander.
+     * Sets the wiretapContentHandler.
      * 
-     * @param wiretapContentHander
-     *            the wiretapContentHander to set
+     * @param wiretapContentHandler
+     *            the wiretapContentHandler to set
      */
-    public void setWiretapContentHander(ContentHandler wiretapContentHander) {
-        this.wiretapContentHander = wiretapContentHander;
+    public void setWiretapContentHandler(ContentHandler wiretapContentHandler) {
```

Comment:
I don't know if anyone would be affected, but this was the only one that popped out to me as possibly a breaking change, since it's a public method

Reply:
All right.
I am waiting for your decision.